index.html

```diff
@@ -159,23 +174,34 @@ <h3>The damage</h3>
         <div class="map">
           <!-- Location Map -->
         </div>
+        <div class="info-text">
+
+          <div class="opening-hours">
+            <h3>Opening hours</h3>
+            <div class="dash-detail"></div>
+            <div class="timetable">
+              <span class="days">Monday - Friday:&nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp9:00 - 21:00</span>
+              <span class="days">Saturday:&nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp 10:00 - 17:00</span>
+              <span class="days">Sunday:&nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp 10:00 - 17:00</span>
+            </div>
+          </div>
 
-        <div class="opening-hours">
-          <h3>Opening hours</h3>
-          <div class="timetable">
-            <span class="days">Mon - Fri</span><span class="hours">9:00 - 21:00</span>
-            <span class="days">Sat - Sun</span><span class="hours">10:00 - 17:00</span>
+          <div class="contact">
+            <h3>Contact</h3>
+            <div class="dash-detail"></div>
+            <span class="contact-number">PHONE: +445895265758</span>
+            <span class="footnote">*Call for bookings and regular enquiries about pricing or services, for anything else you can also contact us on info@londonbarberhood.com</span>
           </div>
         </div>
 
-        <div class="contact">
-          <h3>Contact</h3>
-          <span class="contact-number">+445895265758</span>
-          <span class="footnote">*Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris ornare posuere cursus.</span>
-        </div>
       </div>
 
       <div class="copy">
+        <div class="copy-social">
+          <a href="#" class="fa fa-instagram"></a>
```

Comment:
Recuerda que si coges los SVG de font awesome, te ahorrarás unos cuantos KB de cargar toda la librería y tu página cargará más rápida.

Reply:
lo intenté, y te dan un <i> para pegar en tu html, que en teoría es el icono, y lo metí. La cosa en que no lo veía por ninguna parte. Los busque con devtools y estaban en la pagina pero ponía 0x0 de tamaño... eso que es, height and width? Intente cambiarlos pero no conseguí que se hiciesen mas grandes